### PR TITLE
Resolution docs

### PR DESCRIPTION
**Pull request by mnonnenmacher**
Please see the commit messages for details.

---
This change is [Reviewable](https://reviewable.io/reviews/heremaps/oss-review-toolkit/963)

### CODE REVIEW

File: docs/Configuration.md [hunk 1]
```diff
@@ -147,3 +147,70 @@ they get annotated with the reason and comment from the resolution.
...
+## Global configuration
+
+This section describes repository independent configuration options for ORT.
+
+### Curating metadata of packages
+
+In order to scan the source code of dependencies, ORT relies on the metadata provided by those packages. Often the
+metadata contains information how to locate the source code, but this is not always the case. Often the metadata of
+packages has no VCS information, points to outdated repositories, or the repositories are not correctly tagged. Because
+this information can not always be fixed in remote packages ORT provides a mechanism to curate metadata of packages.
```
Comment (sschuberth) [inline docs/Configuration.md]: in remote packages -> upstream?
comma afterwards

File: docs/Configuration.md [hunk 2]
```diff
@@ -147,3 +147,70 @@ they get annotated with the reason and comment from the resolution.
...
+
+These curations can be configured in a YAML file that has to be passed to the `analyzer`. The data from the curations
```
Comment (sschuberth) [inline docs/Configuration.md]: I'm not sure whether we should be specific to the format here. Maybe just say "configuration file"? Or just "file" as you already say "configured" directly before.

File: docs/Configuration.md [hunk 3]
```diff
@@ -147,3 +147,70 @@ they get annotated with the reason and comment from the resolution.
...
+file will overwrite the metadata provided by the packages themselves. This way it is possible to fix borken VCS URLs or
```
Comment (sschuberth) [inline docs/Configuration.md]: overwrite -> amend

File: docs/Configuration.md [hunk 4]
```diff
@@ -147,3 +147,70 @@ they get annotated with the reason and comment from the resolution.
...
+  curations:
+    vcs:
+      revision: "f864d24ba07cde4b79f16999b1c99bfb240a441e"
+- id: "NPM::ast-traverse:0.1.1"
+  curations:
+    vcs:
+      revision: "73f2b3c319af82fd8e490d40dd89a15951069b0d"
+```
+
+To use the curations file pass it to the `--package-curations-file` option of the `analyzer`:
```
Comment (sschuberth) [inline docs/Configuration.md]: of the -> to the

File: docs/Configuration.md [hunk 5]
```diff
@@ -147,3 +147,70 @@ they get annotated with the reason and comment from the resolution.
...
+
+```
+cli/build/install/ort/bin/ort analyze -i [input-path] -o [analyzer-output-path] --package-curations-file [curations-file-path]
+```
+
+In future we will integrate [ClearlyDefined](https://clearlydefined.io/) as a source for curated metadata. Until then,
```
Comment (sschuberth) [inline docs/Configuration.md]: In the

File: docs/Configuration.md [hunk 6]
```diff
@@ -147,3 +147,70 @@ they get annotated with the reason and comment from the resolution.
 
 For details about the available reasons, see
 [ErrorResolutionReason.kt](../model/src/main/kotlin/config/ErrorResolutionReason.kt).
+
```
Comment (sschuberth) [inline docs/Configuration.md]: Sorry, I saw to late that the docs were just moved. I'll keep my comments for reference.

File: docs/Configuration.md [hunk 7]
```diff
@@ -214,3 +218,27 @@ cli/build/install/ort/bin/ort analyze -i [input-path] -o [analyzer-output-path]
 
 In future we will integrate [ClearlyDefined](https://clearlydefined.io/) as a source for curated metadata. Until then,
 and also for curations for internal packages that cannot be published, the curations file can be used.
+
+### Resolving errors
+
+Global error resolutions work the same way as the project specific error resolutions described
```
Comment (sschuberth) [inline docs/Configuration.md]: works

File: docs/Configuration.md [hunk 8]
```diff
@@ -214,3 +218,27 @@ cli/build/install/ort/bin/ort analyze -i [input-path] -o [analyzer-output-path]
...
+[above](#resolving-errors), but instead of adding them to the `.ort.yml` file of the project, they are added to a
+separate configuration file, which makes them reusable for different projects. Global error resolutions should only be
```
Comment (sschuberth) [inline docs/Configuration.md]: no comma after "file"

File: docs/Configuration.md [hunk 9]
```diff
@@ -214,3 +218,27 @@ cli/build/install/ort/bin/ort analyze -i [input-path] -o [analyzer-output-path]
...
+used for generic errors that can always be ignored. One example where this applies would be scan errors in the
+repositories of third-party packages which affect files which are not built into the package, like documentation or test
```
Comment (sschuberth) [inline docs/Configuration.md]: second "which" -> "that"

File: docs/Configuration.md [hunk 10]
```diff
@@ -214,3 +218,27 @@ cli/build/install/ort/bin/ort analyze -i [input-path] -o [analyzer-output-path]
...
+The structure of the resolutions file is:
+
+```yaml
+errors:
+- message: "A regular expression matching the error message."
+  reason: "One of: BUILD_TOOL_ISSUE|CANT_FIX_ISSUE|SCANNER_ISSUE"
+  comment: "A comment further explaining why the reason above is applicable here."
+```
+
+To resolutions file must be passed to the `--resolutions-file` option of the `reporter`:
```
Comment (sschuberth) [inline docs/Configuration.md]: To -> The
"can be passed using the"

File: docs/Configuration.md (hunk 6 above)
Comment (mnonnenmacher) [reply docs/Configuration.md]: Applied some of them anyway.

**Pull request merged**